.dockerignore

```diff
@@ -1,10 +1,21 @@
 # Prevent vendor directory from being copied to ensure we are not not pulling unexpected cruft from
 # a user's workspace, and are only building off of what is locked by dep.
+*.iml
```

Comment:
while these files are small, they can result in invalidating the Docker caches, forcing build steps to be re-run